packages/cli-kit/src/store.ts

```diff
@@ -44,21 +42,13 @@ let _instance: CLIKitStore | undefined
 
 export function cliKitStore() {
   if (!_instance) {
-    throw new Abort("The CLIKitStore instance hasn't been initialized")
-  }
-  return _instance
-}
-
-export async function initializeCliKitStore() {
-  if (!_instance) {
-    // eslint-disable-next-line require-atomic-updates
     _instance = new CLIKitStore({
```

Comment:
I assume there's no issue if a race conditions causes this to be called twice, right?